component/common/loki/client/client_writeto.go

```diff
@@ -70,7 +59,7 @@ func (c *clientWriteTo) SeriesReset(segmentNum int) {
 	}
 }
 
-func (c *clientWriteTo) AppendEntries(entries wal.RefEntries) error {
+func (c *clientWriteTo) AppendEntries(entries wal.RefEntries, segment int) error {
```

Comment:
segment is unused

Reply:
Yes, but it's part of the interface that the method attests to be a `wal.WriteTo`. Also, in the interface, the segments in the `AppendEntries` is needed for https://github.com/grafana/agent/pull/5590

Reply:
Gotcha, I think  some linkers will pick it up if its the name is not an underscore. I think my local linter did but not what we run for drone.